src/Qir/.clang-format

```diff
@@ -1,37 +1,37 @@
-# https://clang.llvm.org/docs/ClangFormatStyleOptions.html
-
----
-Language: Cpp
-BasedOnStyle: Microsoft
-
-# page width
-ColumnLimit: 120
-ReflowComments: true
-
-# tabs and indents
-UseTab: Never
-IndentWidth: 4
-TabWidth: 4
-AccessModifierOffset: -2
-NamespaceIndentation: Inner
-
-# line and statements layout
-BreakBeforeBraces: Allman
-BinPackParameters: false
-AlignAfterOpenBracket: AlwaysBreak
-AllowShortIfStatementsOnASingleLine: WithoutElse
-AllowShortFunctionsOnASingleLine: Empty
-AllowAllConstructorInitializersOnNextLine: false
-AllowAllArgumentsOnNextLine: true
-AllowAllParametersOfDeclarationOnNextLine: false
-BreakBeforeTernaryOperators: true
-BreakConstructorInitializers: BeforeComma
-
-# misc
-Cpp11BracedListStyle: true
-FixNamespaceComments: true
-IncludeBlocks: Preserve
-SpaceBeforeInheritanceColon : true
-SpaceBeforeParens: ControlStatements
-DerivePointerAlignment: false
-PointerAlignment: Left
+# https://clang.llvm.org/docs/ClangFormatStyleOptions.html
+
+---
+Language: Cpp
+BasedOnStyle: Microsoft
+
+# page width
+ColumnLimit: 120
+ReflowComments: true
+
+# tabs and indents
+UseTab: Never
+IndentWidth: 4
+TabWidth: 4
+AccessModifierOffset: -2
+NamespaceIndentation: Inner
+
+# line and statements layout
+BreakBeforeBraces: Allman
+BinPackParameters: false
+AlignAfterOpenBracket: AlwaysBreak
+AllowShortIfStatementsOnASingleLine: WithoutElse
+AllowShortFunctionsOnASingleLine: Empty
+AllowAllConstructorInitializersOnNextLine: false
+AllowAllArgumentsOnNextLine: true
+AllowAllParametersOfDeclarationOnNextLine: false
+BreakBeforeTernaryOperators: true
+BreakConstructorInitializers: BeforeComma
+
+# misc
+Cpp11BracedListStyle: true
+FixNamespaceComments: true
+IncludeBlocks: Preserve
+SpaceBeforeInheritanceColon : true
+SpaceBeforeParens: ControlStatements
+DerivePointerAlignment: false
+PointerAlignment: Left
```

Comment:
(Up to you)
[FYI A](https://teams.microsoft.com/l/message/19:bafaeb809d7342d68f4ad99cdc1675af@thread.skype/1617134257928?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=e6e22ac8-e11f-44d0-b599-46e9bea42c53&parentMessageId=1617058745524&teamName=Quantum%20Systems&channelName=QIR&createdTime=1617134257928)
[FYI B](https://teams.microsoft.com/l/message/19:bafaeb809d7342d68f4ad99cdc1675af@thread.skype/1617135209545?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=e6e22ac8-e11f-44d0-b599-46e9bea42c53&parentMessageId=1617058745524&teamName=Quantum%20Systems&channelName=QIR&createdTime=1617135209545)

Reply:
See my [comment below](https://github.com/microsoft/qsharp-runtime/pull/598/files#r606389734) to Cesar. For some of these files, the line ending update was intentional so the files are correctly set up to work well cross-platform.